tests/core/pyspec/eth2spec/test/bellatrix/fork_choice/test_should_override_forkchoice_update.py

```diff
@@ -125,7 +125,9 @@ def test_should_override_forkchoice_update__true(spec, state):
     signed_block = state_transition_and_sign_block(spec, state, block)
 
     # Make the head block late
-    attesting_cutoff = spec.config.SECONDS_PER_SLOT // spec.INTERVALS_PER_SLOT
+    # Round up to nearest second
+    attestation_due_ms = spec.get_slot_component_duration_ms(spec.config.ATTESTATION_DUE_BPS)
+    attesting_cutoff = (attestation_due_ms + 999) // 1000
     current_time = state.slot * spec.config.SECONDS_PER_SLOT + store.genesis_time + attesting_cutoff
```

Comment:
Is the reason we keep `SECONDS_PER_SLOTS`  around as deprecated to so that we don't need to change these tests?

Reply:
This config variable is used in a ton of places & I don't want to do too much in a single PR. I would like to handle (remove usages) of `SECONDS_PER_SLOTS` in a follow-up PR. The focus of this PR is `INTERVALS_PER_SLOT`.